src/components/Loader/Loader.vue

```diff
@@ -1,31 +1,37 @@
 <template>
-  <div class="loader" :class="{ hide: hide, secondary }"></div>
+  <div v-if="secondary" class="loader-secondary" :class="{ hide: hide }"></div>
+  <div v-else class="spinner-container" :class="{ hide: hide }">
+    <div class="loader-ripple">
+      <div class="animation-div"></div>
+      <div class="animation-div"></div>
+    </div>
+  </div>
 </template>
 
 <script>
 export default {
   props: {
     noFlickering: { type: Boolean, default: false },
-    secondary: { type: Boolean, default: false },
+    secondary: { type: Boolean, default: false }
   },
   data() {
     return {
-      hide: false,
+      hide: false
```

Comment:
I don't understand it's purpose

Reply:
This was here before my PR. No idea why 🤷‍♂️